Lib/gflanguages/data/languages/bax_Bamu.textproto

```diff
@@ -4,3 +4,12 @@ script: "Bamu"
 name: "Bamun"
```

Comment:
Is consistency in spelling important here?  Bamu**n** vs. Bamu**m**?

Noto uses "Bamum".